Refactor to pyansys standards (#140)
pyansys/pydpf-post: docs/source/user_guide/post_processing.rst

The DPF-Post package provides a Python interface to DPF (Data
Processing Framework) that is streamlined for post-processing.

A central concept in DPF-Post is the :class:`DpfSolution
<ansys.dpf.post.dpf_solution.DpfSolution>` class. It is the entry point for

> browsing the contents of a result file.  It can be obtained as follows:

Reviewer: ```suggestion
browsing the contents of a result file.  You can obtained the contents with:
```
